### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Home Page for my Ecomm App
1.  landing page with a list of featured categories.
2. Navbar 
3. Footer with social handles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+               <button class="btn btn-secondary">Shop Now</button>
+             </div>
+        </div>
+
+        <div class="two-box-section"> 
+           <article class="sub-section-container"> 
+             <div class="sub-section">
+               <div class="sub-section-image">
+                 <img src="/images/Plant1.jpg" alt="" class="ss-image">
+               </div>
```
Comment (Prashantrajput11) [inline index.html]: I liked the fact that you are using consistent color naming convention for css but here i see some alt tags values missing. Rest all looks good.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                <div class="header-icon">
+                   <span class="material-icons">shopping_cart</span>                   
+                </div>
+                <div class="header-icon"> 
+                    <span class="material-icons">favorite_border</span>                   
+                </div>                         
+             </div> 
+        </div>
+    </header>    
+    <section class="main-content-page">
```
Comment (anuj-kumary) [inline index.html:32]: change your class to className

File: homepage/homepage.css [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+.hero-img > img{
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    opacity: 0.7;
+}
+
+.hero-img-overlay{
+    position: absolute;
+    z-index: 999;
```
Comment (rohit8686) [inline homepage/homepage.css]: better not to give this much z index value

File: index.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    <link rel="stylesheet" href="/style.css">
+    <title>Midori | Home</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header-items">
+            <div class="nav-brand">Midori</div>
+              <div class="input-icon icons-left">
+                <span class="material-icons">search</span> 
+                <input class="input" type="" id="" name="" placeholder="search"/>
```
Comment (rohit8686) [inline index.html]: name is empty here, if not necessary better to remove it and rest all looks good

**Review by rohit8686 — APPROVED** [review]

File: homepage/homepage.css [hunk 5]
```diff
@@ -0,0 +1,174 @@
+.card-image{
+    opacity: 0.7;
+}
+
+.category{ 
+    position: relative;  
+    width: 18.75rem;
```
Comment (samyakshah3008) [inline homepage/homepage.css]: You can prefer giving max or min-width for responsiveness, can add later on

**Review by samyakshah3008 — APPROVED** [review]
Few things I liked about this PR: 
Good folder structure
Clean code
Good naming conventions
Proper semantic html tags 

Suggestions: 
You can attach live link and images as for preview 
Can later on add comments where necessary for better code review 

Rest is all good ✅

**Review by RahullRawat — APPROVED** [review]

**Pull request merged**